src/controller/gc/controller.go

```diff
@@ -0,0 +1,208 @@
+package gc
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/goharbor/harbor/src/jobservice/job"
+	"github.com/goharbor/harbor/src/lib/errors"
+	"github.com/goharbor/harbor/src/lib/log"
+	"github.com/goharbor/harbor/src/lib/q"
+	"github.com/goharbor/harbor/src/pkg/scheduler"
+	"github.com/goharbor/harbor/src/pkg/task"
+)
+
+var (
+	// GCCtl is a global garbage collection controller instance
+	GCCtl = NewController()
```

Comment:
Rename it to `Ctl` to keep the same as other controllers.

Reply:
updated.